From 60d329bb2b20a67323152501969ff1db6a43e467 Mon Sep 17 00:00:00 2001
From: Simon Dawson <spdawson@gmail.com>
Date: Mon, 29 Dec 2014 13:53:51 +0100
Subject: [PATCH 1/5] Add stack direction for the AVR32 architecture

The libsigsegv configure step fails to correctly figure out the direction of
stack growth for avr32. This leads to a zero STACK_DIRECTION definition, and
build failures such as the following.

  http://autobuild.buildroot.net/results/a3fe938f9376533b4777d79deb7a2ee83ed5ce33

Signed-off-by: Simon Dawson <spdawson@gmail.com>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 m4/stack-direction.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 00e0445..39c1a28 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -16,7 +16,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       alpha* | \
       arc | \
       arm* | strongarm* | xscale* | \
-      avr | \
+      avr | avr32 | \
       c1 | c2 | c32 | c34 | c38 | \
       clipper | \
       cris | \
-- 
2.1.0

